clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -81,6 +81,12 @@
                 "configuration": "./syntaxes/lang-config.json"
             }
         ],
+        "snippets": [
```

Comment:
Does this mean, the COBOL snippet will always be added and we need not handle it explicitly?

Reply:
yes